### ISSUE TITLE
About Mac OS Sierra

### ISSUE THREAD

**Issue by simongcc** [ivoronin/ArchiveMounter#1]
Hello! I just tried to install in Sierra. It said the APP cannot be used in this OS. Is it not supporting Sierra? (10.12.6)

**Comment by ivoronin** [comment]
Could you try replacing in `/Applications/Archive Mounter.app/Contents/Info.plist`:
```xml
<key>LSMinimumSystemVersion</key>
<string>10.13</string>
```
with
```xml
<key>LSMinimumSystemVersion</key>
<string>10.12</string>
```

**Comment by ivoronin** [comment]
Please also try this build: [Archive Mounter.zip](https://github.com/ivoronin/ArchiveMounter/files/1958016/Archive.Mounter.zip)

**Comment by ivoronin** [comment]
Should be fixed in latest release (v1.3.0)
